README.md

```diff
@@ -34,9 +34,7 @@ Also available for Arch users on the AUR: [https://aur.archlinux.org/packages/ri
 
 On OS X, you can use [Homebrew](http://brew.sh) to install the dependencies:
 
-    $ brew install python3 gawk gnu-sed gmp mpfr libmpc isl zlib expat
-    $ brew tap discoteq/discoteq
-    $ brew install flock
+    $ brew install python3 gawk gnu-sed gmp mpfr libmpc isl zlib expat texinfo flock
```

Comment:
The MacOS now has multiples version and supports two different arch (amd64 and arm64). I am not sure whether the brew commands are consistent across them, but it works on my Ventura on M1.

Reply:
Thank you for your review. ^0^/
The arch I tested is intel x86_64. In addition, according to the information in the link [[1]](https://formulae.brew.sh/formula/flock), both arch should work.

[1]  https://formulae.brew.sh/formula/flock